calculator.js

```diff
@@ -38,8 +36,19 @@ async function getOperation() {
 }
 
 async function getOperand() {
-  let Operand = parseFloat(await rl.question("Input Operand: "));
-  return Operand;
+  let isSelectingOperand = true;
+  
+  while (isSelectingOperand) {
+    let Operand = parseFloat(await rl.question("Input Operand: "));
+    if (!(isNaN(Operand))) {
+    isSelectingOperand = false;
+    return Operand;    
```

Comment:
You should declare a value to hold the user response before the loop, set it inside the loop when the input is valid, then exit the loop and return the value. Either that or do `while (true)` and return inside the loop. I would recommend the former option.